### PR TITLE
SDIT-1776: 🔧 Attempt to simplify mod security processing

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: helm_deploy/prison-api/values.yaml [hunk 1]
```diff
@@ -37,38 +37,28 @@ generic-service:
...
-      SecRuleUpdateTargetById 921110 "!ARGS:/.*comment/"
-      SecRuleUpdateTargetById 921110 "!ARGS:/text/"
-      SecRuleUpdateTargetById 921120 "!ARGS:/.*comment/"
-      SecRuleUpdateTargetById 921120 "!ARGS:/text/"
-      SecRuleUpdateTargetById 932100 "!ARGS:/.*comment/"
-      SecRuleUpdateTargetById 932100 "!ARGS:/text/"
-      SecRuleUpdateTargetById 932105 "!ARGS:/.*comment/"
-      SecRuleUpdateTargetById 932105 "!ARGS:/text/"
-      SecRuleUpdateTargetById 932150 "!ARGS:/.*comment/"
+      # attack-protocol is 921 range
```
Comment (petergphillips) [inline helm_deploy/prison-api/values.yaml:40]: https://github.com/coreruleset/coreruleset/blob/v3.3/master/rules/REQUEST-921-PROTOCOL-ATTACK.conf

File: helm_deploy/prison-api/values.yaml [hunk 2]
```diff
@@ -37,38 +37,28 @@ generic-service:
...
+      SecRuleUpdateTargetByTag "attack-protocol" "!ARGS:/.*comment/"
+      SecRuleUpdateTargetByTag "attack-protocol" "!ARGS:/text/"
+      # attack-rce is 932 range
```
Comment (petergphillips) [inline helm_deploy/prison-api/values.yaml:43]: https://github.com/coreruleset/coreruleset/blob/v3.3/master/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L114

File: helm_deploy/prison-api/values.yaml [hunk 3]
```diff
@@ -37,38 +37,28 @@ generic-service:
...
+      SecRuleUpdateTargetByTag "attack-rce" "!ARGS:/.*comment/"
+      SecRuleUpdateTargetByTag "attack-rce" "!ARGS:/text/"
       SecRuleUpdateTargetById 932150 "!ARGS:/keywords/"
-      SecRuleUpdateTargetById 932150 "!ARGS:/text/"
-      SecRuleUpdateTargetById 933160 "!ARGS:/.*comment/"
-      SecRuleUpdateTargetById 933160 "!ARGS:/text/"
-      SecRuleUpdateTargetById 933210 "!ARGS:/.*comment/"
+      # attack-injection-php is 933 range
```
Comment (petergphillips) [inline helm_deploy/prison-api/values.yaml:47]: https://github.com/coreruleset/coreruleset/blob/v3.3/master/rules/REQUEST-933-APPLICATION-ATTACK-PHP.conf#L58

File: helm_deploy/prison-api/values.yaml [hunk 4]
```diff
@@ -37,38 +37,28 @@ generic-service:
...
+      SecRuleUpdateTargetByTag "attack-injection-php" "!ARGS:/.*comment/"
+      SecRuleUpdateTargetByTag "attack-injection-php" "!ARGS:/text/"
       SecRuleUpdateTargetById 933210 "!ARGS:/.*description/"
-      SecRuleUpdateTargetById 933210 "!ARGS:/text/"
       SecRuleUpdateTargetById 941370 "!ARGS:/.*comment/"
       SecRuleUpdateTargetById 941370 "!ARGS:/text/"
+      # attack-sqli is 942 range
```
Comment (petergphillips) [inline helm_deploy/prison-api/values.yaml:53]: https://github.com/coreruleset/coreruleset/blob/v3.3/master/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L57

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**